templates/common/jest.config.js

```diff
@@ -1,3 +1,7 @@
+// force timezone to UTC to allow tests to work regardless of local timezone
+// generally used by snapshots, but can affect specific tests
+process.env.TZ = 'UTC';
```

Comment:
I'm wondering if it would be easier for developers to find and maybe adjust the value if we put it in the package.json test script instead. e.g. `"test": "TZ=UTC yarn jest"`. WDYT?

Reply:
from what i could tell vscode was not running `yarn test` and would use the local timezone, you'd have to launch vscode with the env var set.

i saw this when when i tried a remote dev environment over ssh that had UTC, and my local machine was EDT, and then in the drone/ci builds it was in UTC, but the snapshots were EDT.

not 100% sure where it should go.

Reply:
I remember fixing this the same way jack suggests but I don't use vscode and it does make sense that vscode launches the jest process on its own not using scripts.